common/src/main/scala/explore/components/Tile.scala

```diff
@@ -58,58 +56,69 @@ object Tile {
 
   implicit val rawContainerReuse: Reusability[raw.ReactDOM.Container] = Reusability.always
 
+  class Backend() {
+
+    private val infoRef = Ref.toVdom[html.Element]
+
+    def render(p: Props) = {
+      val maximizeButton =
+        Button(
+          as = <.a,
+          basic = true,
+          compact = true,
+          clazz = ExploreStyles.TileStateButton |+| ExploreStyles.BlendedButton,
+          onClick = p.sizeStateCallback(TileSizeState.Maximized)
+        )(Icons.Maximize.fitted(true))
+
+      val minimizeButton =
+        Button(
+          as = <.a,
+          basic = true,
+          compact = true,
+          clazz = ExploreStyles.TileStateButton |+| ExploreStyles.BlendedButton,
+          onClick = p.sizeStateCallback(TileSizeState.Minimized)
+        )(Icons.Minimize.fitted(true))
+
+      <.div(ExploreStyles.Tile)(
+        <.div(
+          ExploreStyles.TileTitle,
+          p.back.map(b => <.div(ExploreStyles.TileButton, b.body)),
+          Menu(
+            attached = MenuAttached.Top,
+            compact = true,
+            borderless = true,
+            secondary = true,
+            clazz = ExploreStyles.TileTitleMenu,
+            tabular = MenuTabular.Right
+          )(
+            MenuItem(as = <.a)(p.title)
+          ),
+          <.span(ExploreStyles.TileTitleInfo,
+                 ^.untypedRef := infoRef,
+                 ^.key := s"tile-info-${p.title}",
+                 <.span
```

Comment:
We can probably remove this inner `span` too.

Reply:
I wasn't sure what was that used for, I'll remove it in the next PR